components/permit-holders/GuardianInformationCard.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box, Text, Divider, VStack, Button } from '@chakra-ui/react'; // Chakra UI
+import PermitHolderInfoCard from '@components/internal/PermitHolderInfoCard'; // Custom Card Component
+import { Guardian } from '@lib/graphql/types'; // Guardian type
+
+type GuardianInformationProps = {
+  guardian: Guardian;
+};
+
+export default function GuardianInformationCard(props: GuardianInformationProps) {
+  const { guardian } = props;
+  return (
+    <PermitHolderInfoCard
+      colSpan={7}
+      header={`Guardian's Information`}
+      editModal={
+        //   TODO: Create edit guardian modal post-mvp
+        <Button color="primary" variant="ghost" textDecoration="underline">
+          <Text textStyle="body-bold">Edit</Text>
+        </Button>
```

Comment:
I think we should just remove this button temporarily since it doesn't do anything